ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -954,11 +960,32 @@ function scatterProcessData(
             processedData.push({ x, y });
             processedXData.push(x);
             processedYData.push(y);
+            if (xRangeNeeded) {
+                xDataMax = Math.max(xDataMax, x);
+                xDataMin = Math.min(xDataMin, x);
+            }
+            if (yRangeNeeded) {
+                yDataMax = Math.max(yDataMax, y);
+                yDataMin = Math.min(yDataMin, y);
+            }
         } else {
             cropped = true;
         }
     }
 
+    if (xRangeNeeded) {
+        xAxis.setOptions({
```

Comment:
Is there any reason to run `setOptions`? It is cheaper to just set the properties directly, and as far as I can see this works. Also taking into consideration that what we are really looking for is `dataMin` and `dataMax`:

```ts
    if (xRangeNeeded) {
        xAxis.dataMax = xDataMax;
        xAxis.dataMin = xDataMin;
    }
```